build/cluster-operator-ansible/playbooks/cluster-api-prep/deploy-cluster-api.yaml

```diff
@@ -76,19 +76,20 @@
         cluster_api_key: "{{ clusterapi_key.content }}"
         cluster_api_ca_bundle: "{{ clusterapi_crt.content }}"
 
+  - name: create temp file to hold the template
+    tempfile:
+      state: file
+    register: temp_file
+
+  - name: copy template over
+    copy:
+      src: files/cluster-api-template.yaml
+      dest: "{{ temp_file.path }}"
```

Comment:
For future consideration we could leave this as a persistent file on the host, then we would accurately see if something changed of not as a result of the copy and could do changed_when: false on the oc process command.